app/Option.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
Why do you have the Option Model here already? It should be on a different branch. This branch is about creating a new Admin.